### ISSUE TITLE
how to run WarfaceBot on Iphone

### ISSUE THREAD

**Issue by I-LOVE-C2H5OH** [Levak/warfacebot#508]
1) install TestFlight
2) install ISH
3) install libs (git, gcc, zlib-dev, openssl-dev, curl, readline-dev, make, rust-stdlib)(apk add "LIB")
4) write git clone https://github.com/Levak/warfacebot.git
5) cd warfacebot
6) write "make" and wait
7) edit wb.sh (vi wb.sh) add rm QueryCache/*
8) write sh wb.sh server
9) write login and password
the bot is running
![image](https://user-images.githubusercontent.com/41304540/88998641-5067ad80-d314-11ea-8116-16a36d919303.png)

**Comment by Lako-FC** [comment]
nice

**Comment by I-LOVE-C2H5OH** [comment]
> отлично

thx

**Comment by Levak** [comment]
> edit wb.sh (vi wb.sh) add rm QueryCache/*

Any reason why removing cache is mandatory on iPhone?
PS: Use `rm -rf QueryCache` instead (or `rm -f QueryCache/*`) as it will error the first time you run it otherwise (no such file or directly). `-f` flag will force deletion even if no file exist.

**Comment by I-LOVE-C2H5OH** [comment]
> > отредактировать wb.sh (vi wb.sh) добавить rm QueryCache / *
> 
> Есть ли причина, по которой удаление кеша на iPhone обязательно?
> PS: Используйте `rm -rf QueryCache`вместо (или `rm -f QueryCache/*`), поскольку в противном случае при первом запуске будет ошибка (нет такого файла или напрямую). `-f`флаг принудительно удалит, даже если файл не существует.

Yes, the behavior is as if the keys are not suitable, but at the first launch it starts, and the subsequent ones do not
